src/webui/www/private/filters.html

```diff
@@ -60,9 +67,44 @@
         }
     });
 
+    var tagsFilterContextMenu = new TagsFilterContextMenu({
```

Comment:
you should use `const` or `let` whenever possible in place of `var`, this applies to everywhere else too.